BUILDING.txt

```diff
@@ -143,20 +143,17 @@ that the viewer has already been built, per above.)
   > ./configure --with-pic --without-dtrace --disable-static --disable-dri \
       --disable-xinerama --disable-xvfb --disable-xnest --disable-xorg \
       --disable-dmx --disable-xwin --disable-xephyr --disable-kdrive \
-      --disable-config-dbus --disable-config-hal --disable-config-udev \
-      --disable-dri2 --enable-install-libxf86config --enable-glx \
+      --disable-config-hal --disable-config-udev --disable-dri2 --enable-glx \
       --with-default-font-path="catalogue:/etc/X11/fontpath.d,built-ins" \
-      --with-fontdir=/usr/share/X11/fonts \
       --with-xkb-path=/usr/share/X11/xkb \
       --with-xkb-output=/var/lib/xkb \
       --with-xkb-bin-directory=/usr/bin \
-      --with-serverconfig-path=/usr/lib[64]/xorg \
-      --with-dri-driver-path=/usr/lib[64]/dri \
```

Comment:
Removing this seems wrong. It needs to match your build of mesa.

Reply:
We removed them since these flags no longer exists for configure on RHEL 9.

Reply:
Starting from xorg v. 1.10, the dri is found automatically. Since TigerVNC requires xorg v. 1.16, these paths can safely be removed.